Source Control Dashboard Automation

Jitendra-Deshmukh opened the pull request:
#25 Source code needs to be dumped along with the excel sheet - Added

addin to export/Import source code.

Jitendra-Deshmukh committed:
Source Control Dashboard Automation

#25 Source code needs to be dumped along with the excel sheet - Added
addin to export/Import source code.

alokmodak commented:
Can you please add some details on how the addin will work?
Can you please explain the mechanism, how the auto dump and auto reload works?
Where is the source code for functions like "FCO_Calculations"?

alokmodak commented:
ok just saw that the dumped code is also checked in. Thanks.
Can you please explain the mechanism, how the auto dump and auto reload works?

Jitendra-Deshmukh committed:
Dashboard Automation Update

#48 Remove sharepoint - Rmoved
#47 Remove MAT, MQT options on UI - Removed
#46 Change "Dashboard" to "Operating Review" - Changed over UI
#44 FCO output not correct- Corrected
#43 Have consistent font  - text size set to 18 for all.
